app/code/Magento/QuoteGraphQl/Model/Resolver/CartPrices.php

```diff
@@ -73,7 +73,7 @@ private function getAppliedTaxes(Total $total, string $currency): array
         $appliedTaxesData = [];
         $appliedTaxes = $total->getAppliedTaxes();
 
-        if (count($appliedTaxes) === 0) {
+        if ($appliedTaxes === null || count($appliedTaxes) === 0) {
```

Comment:
``` suggestion
        if (!empty($appliedTaxes)) {
```

Reply:
I assume you meant `empty` instead of `!empty`?

Reply:
Sure, you're right! 👍